### PR TITLE
added Stack class

### PR DESCRIPTION
**Pull request by garimagupta1507**

### CODE REVIEW

**Comment by garimagupta1507** [comment]
pop:7
pop:2
pop:6
peek :5

File: src/main/java/Stack.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+public class Stack {
+    ListNode head;
```
Comment (agupta0905) [inline src/main/java/Stack.java]: write a constructor initialize head to null

File: src/main/java/Stack.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    public void push(int element){
+        ListNode tmp = new ListNode();
+        tmp.val = element;
+        tmp.next = head;
+        head = tmp;
+    }
+
+    public int pop(){
```
Comment (agupta0905) [inline src/main/java/Stack.java:15]: Edge case when head is null missed

**Review by agupta0905 — COMMENTED** [review]
Added my comments

**Review by agupta0905 — APPROVED** [review]
r=me

**Pull request merged**